src/routes/index.js

```diff
@@ -1,5 +1,10 @@
 const express = require("express");
 
+const { mypageRouter } = require("./mypageRouter");
+
 const routes = express.Router();
 // try-catch문에 대해서 err catch 해주는 부분이 라우터 단에서 필요하다.
+
+routes.use("/mypage", mypageRouter);
+
```

Comment:
```suggestion
const userHealthInfoRouter = require("./userHealthInfoRouter");

const routes = express.Router();
// try-catch문에 대해서 err catch 해주는 부분이 라우터 단에서 필요하다.

routes.use("/userHealthInfo", userHealthInfoRouter);

```
- router, controller, service, models는 우리가 주로 다루는 데이터(자원, resource, entity)단위로 구분해 주시는 것이 좋습니다.
- Entity 가운데 mypage라는 것은 없고, 사실상 유저 정보라서, user와 관련된 네이밍을 추천드립니다.
- `users` Entity 관련 데이터라고 하기엔 행정적인 사용자 정보가 아닌, 서비스적인 사용자 정보를 다루는 API로 보이네요. 많이 고민하다가 userHealthInfo 라는 Entity 명을 추천드립니다. 변경하셔도 좋습니다 :)

Reply:
소헌님 질문이 있습니다. 이렇게 라우터 명을 변경하는 경우에는. 그동안 mypageRouter, mypageController, mypageService, mypageDao를 모두 
userHealthInfo____으로 변경해야 하나요?

Reply:
@kroonstazy 네 그렇습니다! :) 한세트로 변경해주시면 됩니다!